types/color/index.d.ts

```diff
@@ -1,101 +1,132 @@
-// Type definitions for color 1.0
-// Project: https://www.npmjs.com/package/color
-// Definitions by: Karol Janyst <https://github.com/LKay>
+// Type definitions for color 2.0
+// Project: https://github.com/qix-/color#readme
+// Definitions by: Junyoung Clare Jang <https://github.com/Airlun>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
-declare namespace Color {
+import convert = require('color-convert');
+
+type ColorParam = Color | string | ArrayLike<number> | number | { [key: string]: any };
 
-    type ColorParam = string | { [param: string]: number } | number;
+declare class Color {
```

Comment:
This is incorrect representation of this library as you the constructors are essentially call signatures (https://github.com/Qix-/color#constructors) Since this doesnt mention that the constructors need to follow new Color you cannot declare Color as a class